src/vma/dev/cq_mgr.h

```diff
@@ -305,4 +304,34 @@ class cq_mgr
 // Since we have a single TX CQ comp channel for all cq_mgr's, it might not be the active_cq object
 cq_mgr* get_cq_mgr_from_cq_event(struct ibv_comp_channel* p_cq_channel);
 
+#ifdef HAVE_INFINIBAND_MLX5_HW_H
+
+class cq_mgr_mlx5: public cq_mgr
+{
+public:
+	cq_mgr_mlx5(ring_simple* p_ring, ib_ctx_handler* p_ib_ctx_handler, uint32_t cq_size, struct ibv_comp_channel* p_comp_event_channel, bool is_rx);
+	virtual ~cq_mgr_mlx5();
+
+	virtual inline mem_buf_desc_t*		poll(uint32_t&	opcode, uint32_t&	status);
```

Comment:
#### STYLE

1. The indentation here seems completely random. Please sort it out.
2. Please make sure there are no tabs between function parameters (as in before `opcode` and `status` above).